hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -269,6 +269,14 @@
 	private final Map subclassPropertyAliases = new HashMap();
 	private final Map subclassPropertyColumnNames = new HashMap();
 
```

Comment:
I've added these warnings because I know that there can be problems with duplicated property names. I've looked at current usage, and I don't see anything that gets broken except for the bug that is fixed by this pull request.

Should I open a Jira to make changes to PropertyMapping SPI to deal with this, or is this already covered somehow?